lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts

```diff
@@ -54,21 +48,10 @@ export class TaskCloudService extends BaseCloudService {
      */
     completeTask(appName: string, taskId: string): Observable<TaskDetailsCloudModel> {
         if (appName && taskId) {
-            const queryUrl = this.buildCompleteTaskUrl(appName, taskId);
-            const bodyParam = { 'payloadType': 'CompleteTaskPayload' };
-            const pathParams = {}, queryParams = {}, headerParams = {},
-                formParams = {}, contentTypes = ['application/json'], accepts = ['application/json'];
+            const url = `${this.getBasePath(appName)}/rb/v1/tasks/${taskId}/complete`;
+            const payload = { 'payloadType': 'CompleteTaskPayload' };
 
-            return from(
-                this.apiService
-                    .getInstance()
-                    .oauth2Auth.callCustomApi(
-                        queryUrl, 'POST', pathParams, queryParams,
-                        headerParams, formParams, bodyParam,
-                        contentTypes, accepts, null, null)
-            ).pipe(
-                catchError((err) => this.handleError(err))
```

Comment:
are we catching these errors at any point or we're just trusting these endpoints to the point that we don't need them anymore?